website/src/css/_config.scss

```diff
@@ -0,0 +1,348 @@
+  --#{$prefix}-color-a2: #171b20;
+  --#{$prefix}-color-a3: #202429;
+  --#{$prefix}-color-a4: #313439;
+  --#{$prefix}-color-a5: #575a5d;
+}
+
+:root {
+  @include detox-colors('detox-color');
+  @include ifm-colors('ifm');
+
+  &[data-theme='dark'] {
+    @include detox-colors-dark('detox-color');
+  }
+}
+
+// Fonts
+
+@mixin h1-80 {
+  font-family: 'Nunito Sans';
+  font-size: 80px;
+  font-weight: 400;
+}
+
+@mixin h1-96 {
+  font-family: 'Nunito Sans';
+  font-size: 96px;
+  font-weight: 400;
+}
+
+@mixin h2-58 {
+  font-family: 'Nunito Sans';
+  font-size: 58px;
+  font-weight: 700;
+}
+
+@mixin h2-70 {
+  font-family: 'Nunito Sans';
+  font-size: 70px;
+  font-weight: 700;
+}
+
+@mixin h3-40 {
+  font-family: 'Nunito Sans';
+  font-size: 40px;
+  font-weight: 600;
+}
+
+@mixin h3-50 {
+  font-family: 'Nunito Sans';
+  font-size: 50px;
+  font-weight: 600;
+}
+
+@mixin h4-30 {
+  font-family: 'Nunito Sans';
+  font-size: 30px;
+  font-weight: 700;
+}
+
+@mixin h4-40 {
+  font-family: 'Nunito Sans';
+  font-size: 40px;
+  font-weight: 700;
+}
+
+@mixin h5-24 {
+  font-family: 'Nunito Sans';
+  font-size: 24px;
+  font-weight: 400;
+}
+
+@mixin h5-30 {
+  font-family: 'Nunito Sans';
+  font-size: 30px;
+  font-weight: 400;
+}
+
+@mixin t1-20 {
+  font-family: 'Nunito Sans';
+  font-size: 20px;
+  font-weight: 400;
+}
+
+@mixin t1-30 {
+  font-family: 'Nunito Sans';
+  font-size: 30px;
+  font-weight: 400;
+}
+
+@mixin t2-16 {
+  font-family: 'Nunito Sans';
+  font-size: 16px;
+  font-weight: 400;
+}
+
+@mixin t2-26 {
+  font-family: 'Nunito Sans';
+  font-size: 26px;
+  font-weight: 400;
+}
+
+@mixin t3-13 {
+  font-family: 'Nunito Sans';
+  font-size: 13px;
+  font-weight: 400;
+}
+
+@mixin t3-20 {
+  font-family: 'Nunito Sans';
+  font-size: 20px;
+  font-weight: 400;
+}
+
+// Showcase buttons
+
+@mixin showcase-default-btn {
+  background: radial-gradient(103.29deg, #7633e6 -14.83%, #5d71e6 102.26%);
+  border-radius: 10px;
+  width: 217px;
+  height: 50px;
+  font-family: 'Nunito Sans';
+  font-style: normal;
+  font-size: 16px;
+  font-weight: 100;
+  line-height: 25px;
+  color: #ffffff;
+  text-decoration: none;
+
+  &:hover {
+    background: #7633e5;
+    color: #ffffff;
+    text-decoration: none;
+  }
+
+  &:active {
+    background: #5c25af;
+    color: #ffffff;
+    text-decoration: none;
+  }
+}
+
+@mixin showcase-store-btn {
+  width: 115px;
+  height: 30px;
+  line-height: 30px;
+  color: #ffffff;
+  font-size: 16px;
+  font-family: 'Nunito Sans';
+  font-style: normal;
+  font-weight: 300;
+  text-align: center;
+  text-decoration: none;
+  border-radius: 20px;
+  background: rgba(255, 255, 255, 0.06);
+
+  &:hover {
+    background: #7633e5;
+    color: #ffffff;
+    text-decoration: none;
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+  }
+
+  &:active {
+    background: #5c25af;
+    color: #ffffff;
+    text-decoration: none;
+  }
+}
+
+// Social buttons
+
+@mixin discord-btn {
```

Comment:
Very doubtful that it should be a mixin. The same goes for twitter-btn, etc.

Reply:
Yeah, that's true. They aren't being used anywhere else. Thanks!